src/main/java/net/dv8tion/jda/api/entities/Invite.java

```diff
@@ -310,12 +310,17 @@ default String getUrl()
     boolean isExpanded();
 
     /**
-     * Whether this Invite grants only temporary access or not
+     * Whether this Invite grants only temporary access or not.
+     *
+     * <p>This works only for expanded invites and will throw a {@link IllegalStateException} otherwise!
```

Comment:
This line is pointless. The entire point of the `@throws` block is to explain possible exceptions which it already does

Reply:
I did that for consistency's sake, as other methods have that too.